### DISCUSSION TITLE
Direct Messager Discussion

### DISCUSSION THREAD

**Discussion by GitUser1795** [GitUser1795/Direct-Messager#1] [General]
## 👋 Welcome!
I had a friend set up a discussion, because I don't know how to handle issues. If someone wants to ask anything or wants to add something, just send a message below and I will communicate with you.

**Comment by PhantomXL** [comment]
Hello. I am interested in contributing to the project. Should I make an issue or just chat through here and do a pull request when necessary?

**Reply by GitUser1795** [reply]
Yeah. That would be great, in case someone else sees it and wants to contribute too. I'll just check your code through the pull request a little bit later in the day.

**Reply by PhantomXL** [reply]
I sent the pull request and I'm waiting for your response.
